Terminal.Gui/ConsoleDrivers/CursesDriver/ClipboardImpl.cs

```diff
@@ -5,11 +5,10 @@ namespace Terminal.Gui;
 
 /// <summary>A clipboard implementation for Linux. This implementation uses the xclip command to access the clipboard.</summary>
 /// <remarks>If xclip is not installed, this implementation will not work.</remarks>
-class CursesClipboard : ClipboardBase
+internal class CursesClipboard : ClipboardBase
```

Comment:
I don't think the code cleanup engine is allowed to separate things to one type per file, but I'm noting this file has multiple types.

I'll look for that in the settings, too, when I'm looking for the type check operator refactoring setting.

Reply:
I didn't see anything that would enable this for automatic code cleanup, so it's just something we'll have to invoke wherever the analyzer complains about it. Oh well.